ksp_plugin/flight_plan.hpp

```diff
@@ -113,55 +140,58 @@ class FlightPlan {
       serialization::FlightPlan const& message,
       not_null<Ephemeris<Barycentric>*> ephemeris);
 
-  static std::int64_t constexpr max_ephemeris_steps_per_frame = 1000;
+  static constexpr std::int64_t max_ephemeris_steps_per_frame = 1000;
+
+  static constexpr Error bad_desired_final_time = Error::INVALID_ARGUMENT;
+  static constexpr Error does_not_fit = Error::INVALID_ARGUMENT;
+  static constexpr Error singular = Error::INVALID_ARGUMENT;
```

Comment:
The fact that these three are equal is going to lead to a lot of confusion, especially since we test for specific statuses in the interface.

`does_not_fit` and `bad_desired_final_time` are both `FAILED_PRECONDITION`-like (they are about the compatibility of the argument with the flight plan, rather than the argument being intrinsically bad like `manœuvre.IsSingular()`).
Make one of them `OUT_OF_RANGE` and we will have distinct statuses.

Reply:
Changed the first two to be `OUT_OF_RANGE`.